docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
As  we do not have alternative to "restart" verb at this moment and we definitely want to have a doc for Cockpit I am merging this.